lib/matplotlib/animation.py

```diff
@@ -309,18 +310,17 @@ def frame_size(self):
         return int(w * self.dpi), int(h * self.dpi)
 
     def _adjust_frame_size(self):
+        log = logging.getLogger(__name__)
         if self.codec == 'h264':
             wo, ho = self.fig.get_size_inches()
             w, h = adjusted_figsize(wo, ho, self.dpi, 2)
             if not (wo, ho) == (w, h):
                 self.fig.set_size_inches(w, h, forward=True)
-                verbose.report('figure size (inches) has been adjusted '
-                               'from %s x %s to %s x %s' % (wo, ho, w, h),
-                               level='helpful')
+                log.info('figure size (inches) has been adjusted '
+                               'from %s x %s to %s x %s' % (wo, ho, w, h))
         else:
             w, h = self.fig.get_size_inches()
-        verbose.report('frame size in pixels is %s x %s' % self.frame_size,
-                       level='debug')
+        log.info('frame size in pixels is %s x %s' % self.frame_size)
```

Comment:
Did you mean to change the lever on this one (makes sense to match the one just above)